help_function/help_function.xml

```diff
@@ -6,6 +6,7 @@
     <test testName="array_copy" configFile="config/TEMPLATE_help_function.xml" />
     <test testName="assign_device_vector" configFile="config/TEMPLATE_help_function.xml" />
     <test testName="move_device_vector" configFile="config/TEMPLATE_help_function.xml" />
+    <test testName="swap_device_vector" configFile="config/TEMPLATE_help_function.xml" />
```

Comment:
Since the tests are using USM, should all of the affected tests use the USM configFile?

Reply:
Yes, fixed.